ee_tests/oc-get-project-logs.sh

```diff
@@ -61,4 +61,8 @@ then
   fi
 fi
 
+buildName=$(oc get builds -n $1 | awk '{print $1}' | tail -1)
+echo ---------- Get stack analysis URL for $buildName ---------------
+oc get build/$buildName -n $1 -o yaml | grep "fabric8.io/bayesian.analysisUrl"
+
```

Comment:
Why don't we print the whole yaml - we might find some relevant information there sometime.